include/irr/asset/ICPUBuffer.h

```diff
@@ -99,6 +99,13 @@ template<
 >
 class CCustomAllocatorCPUBuffer;
 
+//! Specialization of ICPUBuffer capable of taking custom allocators
+/*
+    Take a look that with this usage you have to specify custom alloctor
+    passing an object type for allocation and a pointer to allocated
+    data for it's storage by ICPUBuffer, more control is provided.

```

Comment:
a common trick for creating an `ICPUBuffer` over an already existing `void*` array without any memcpy or taking over the memory ownership is to use a `CCustomAllocatorCPUBuffer` with a `null_allocator` that adopts memory (it is a bit counter intuitive because adopt=take ownership, but a null allocator doesn't do anything, even free the memory, so you're all good)

Reply:
adjusted in following commit